### PR TITLE
Added Simplenote version 2.15.0.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24329)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
#24120 #24119 #24118 #24116 #24114 #24113 #24110 #24109 #24108 #24106 Why? Is there any issue with pipelines?

**Comment by palenshus** [comment]
This is failing due to a SignatureSha256 hash mismatch

**Comment by vedantmgoyal2009** [comment]
> This is failing due to a SignatureSha256 hash mismatch, winget-create will help you avoid these

I took the hash from `winget hash -f`.

File: manifests/a/Automattic/Simplenote/2.15.0/Automattic.Simplenote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- silent
+Installers:
+- Platform:
+  - Windows.Universal
+  Architecture: x64
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.15.0/Simplenote-win-2.15.0-x64.appx
+  InstallerSha256: CB2F977B2EEC83B00588DF81ED1AF883CA26C170262B59F452F88655435A90EF
+  SignatureSha256: 0B949D4FBC8FE97553562F2678E86A2CAA6AF8FBDA26CA954A6AE4DF785AA653
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.15.0/Automattic.Simplenote.installer.yaml:16]:
```suggestion
  SignatureSha256: E78161A416905A639955D8DFAA8AB3FE6CCF0917CBAB388214D7C4D90324B41B
```

File: manifests/a/Automattic/Simplenote/2.15.0/Automattic.Simplenote.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  PackageFamilyName: simplenote_cdagv53c87npa
+  UpgradeBehavior: install
+- Platform:
+  - Windows.Universal
+  Architecture: x86
+  InstallerType: appx
+  Scope: user
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.15.0/Simplenote-win-2.15.0-ia32.appx
+  InstallerSha256: CFD0ED3A0E3C366FF62E5601F532A300E3055C3A22CBFAC574658D61CBD4F9D4
+  SignatureSha256: 01C0122D12D272C021C1E0A50B91FC8BF8CD6CF23007752C5F50EC3C2267B907
```
Comment (ItzLevvie) [inline manifests/a/Automattic/Simplenote/2.15.0/Automattic.Simplenote.installer.yaml:26]:
```suggestion
  SignatureSha256: 87C306F0D5C4575E8D8CF5BBA2F5361EBDFB244A7586D57729ACCF8E02D38BD5
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**